<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceGeneratorDlg</class>
 <widget class="QDialog" name="ServiceGeneratorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Holy service generator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Create holy service between dates</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateFrom">
       <property name="alignment">
        <set>Qt::AlignRight</set>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>and</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateTo">
       <property name="alignment">
        <set>Qt::AlignRight</set>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>563</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="everyNthWeek">
       <property name="text">
        <string>Every</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="everyNthSpin">
       <property name="toolTip">
        <string>1 means every week, 2 every second week ...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>-th week on</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="weekDayCombo1">
       <property name="toolTip">
        <string>Day in week (Monday...)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>beginning on</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="startOfCountingDate">
       <property name="toolTip">
        <string>Set the first date</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="nthWeekdayInMonth">
       <property name="text">
        <string>On</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="weekdayOrder">
       <property name="toolTip">
        <string>Every second monday in month a.s.o.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>-th</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="weekDayCombo2">
       <property name="toolTip">
        <string>Day in the week</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>in the month</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="1" column="2" colspan="2">
      <widget class="QComboBox" name="priestCombo"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>with intention</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>to</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>from</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="4">
      <widget class="QLineEdit" name="intentionEdit"/>
     </item>
     <item row="0" column="3">
      <widget class="QTimeEdit" name="timeTo">
       <property name="toolTip">
        <string>Finish time of the holy service</string>
       </property>
       <property name="displayFormat">
        <string>H:mm</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QComboBox" name="churchCombo"/>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>in the church</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QComboBox" name="clientCombo"/>
     </item>
     <item row="0" column="1">
      <widget class="QTimeEdit" name="timeFrom">
       <property name="toolTip">
        <string>Start time of holy service</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight</set>
       </property>
       <property name="displayFormat">
        <string>H:mm</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>celebrated by</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>for client</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>563</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="usualServicesRadio">
       <property name="text">
        <string>usual services for church</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="UsualChurchCombo" name="usualChurchCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="toolTip">
        <string>Starts the generator</string>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
       <property name="icon">
        <iconset resource="HolyServices.qrc">
         <normaloff>:/images/exec.png</normaloff>:/images/exec.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="skipCollisionRadio">
       <property name="toolTip">
        <string>Generator does NOT overwrite already existing service</string>
       </property>
       <property name="text">
        <string>Skip colliding</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="replaceCollisionRadio">
       <property name="toolTip">
        <string>Generator overwrites already existing services if they happen to be in the same church in the same time</string>
       </property>
       <property name="text">
        <string>Replace colliding</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="format">
      <string notr="true">%p% (%v / %m)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="undoButton">
       <property name="toolTip">
        <string>Revert recent generator run</string>
       </property>
       <property name="text">
        <string>Undo</string>
       </property>
       <property name="icon">
        <iconset resource="HolyServices.qrc">
         <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="redoButton">
       <property name="text">
        <string>Redo</string>
       </property>
       <property name="icon">
        <iconset resource="HolyServices.qrc">
         <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="toolTip">
      <string>Close generator without generating the services</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UsualChurchCombo</class>
   <extends>QComboBox</extends>
   <header>UsualChurchCombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dateFrom</tabstop>
  <tabstop>dateTo</tabstop>
  <tabstop>everyNthWeek</tabstop>
  <tabstop>everyNthSpin</tabstop>
  <tabstop>weekDayCombo1</tabstop>
  <tabstop>startOfCountingDate</tabstop>
  <tabstop>nthWeekdayInMonth</tabstop>
  <tabstop>weekdayOrder</tabstop>
  <tabstop>weekDayCombo2</tabstop>
  <tabstop>timeFrom</tabstop>
  <tabstop>timeTo</tabstop>
  <tabstop>churchCombo</tabstop>
  <tabstop>priestCombo</tabstop>
  <tabstop>clientCombo</tabstop>
  <tabstop>intentionEdit</tabstop>
  <tabstop>usualServicesRadio</tabstop>
  <tabstop>usualChurchCombo</tabstop>
  <tabstop>generateButton</tabstop>
  <tabstop>skipCollisionRadio</tabstop>
  <tabstop>replaceCollisionRadio</tabstop>
  <tabstop>undoButton</tabstop>
  <tabstop>redoButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="HolyServices.qrc"/>
 </resources>
 <connections/>
</ui>
